src/pyfmi/master.pyx

```diff
@@ -1519,15 +1522,14 @@ cdef class Master:
         res = {}
         #Load data
         for i,model in enumerate(self.models):
-            if opts["result_handling"] == "file" or opts["result_handling"] == "binary":
-                stored_res = self.models_dict[model]["result"].get_result()
-                res[i] = AssimuloSimResult(model, self.models_dict[model]["result"].file_name, None, stored_res, None)
-                res[model] = res[i]
-            elif opts["result_handling"] == "none":
-                res[model] = None
-            else:
-                raise fmi.FMUException("Currently only writing result to file/binary or none is supported.")
-        
+            stored_res = self.models_dict[model]["result"].get_result()
+            try:
```

Comment:
Not sure I understand the purpose here. 
Just a safety measure? Any adverse effects if `file_name `does in fact become an empty string?

Reply:
No adverse effect, it will happen for dummy result handlers and when result is stored in memory